### PR TITLE
reorg doc, remove note styling

### LINKED ISSUE #955: Applying Upstream Updates Doc Update
Re: [Applying Upstream Updates](https://pantheon.io/docs/articles/sites/code/applying-upstream-updates)

## Issue Description:

Doc is scary when you first open it

## Suggested Resolution 

Can we get "Apply a Core Update" and image above the fold and move all the warnings and stuff somewhere else?

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #955 

This PR:
- moves essential information above the fold
- removes note treatment from general information that doesn't need to be called out
- removes unnecessary screenshot 

@bmackinney @ari-gold Please review.

### CODE REVIEW

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 1]
```diff
@@ -40,29 +26,36 @@ If you have overwritten core, see [Undo Git Commits](/docs/articles/local/undo-g
 5. Deploy the upstream updates to your Test Environment by clicking **Pull (Content from Live and) Code from Development** in the Code workspace in the Test Environment.
 6. Click **Visit Testing Environment** to test the update against your Live site’s content base.
 7. Deploy the upstream updates to your Live Environment by clicking **Pull Code from Testing** in the Code workspace in the Live environment.
-![The Code tab in a Pantheon site's dashboard's Live environment](/source/docs/assets/images/desk_images/357432.png)
 8. Click **Visit Live Environment** to verify the update is live.
 ![The Code tab in the Pantheon site's dashboard's Live Environment, showing the upstream commits in the log as deployed.](/source/docs/assets/images/desk_images/357435.png)
 
+## About Core Updates
```
Comment (rachelwhitton) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: The user reads 1. how to prepare for automatic updates, 2. how to complete automatic updates, and 3. definition for automatic updates  - seems a bit strange to define the task after completing it. 

I agree we should make it less visually intimidating, but IMO this doesn't seem like an ideal content flow.

Comment (nataliejeremy) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: @ari-gold submitted this issue. I think the name of the page defines the task and it should be the priority. There was a lot of content, along with a "note" and "warning" above it that made it seem like completing the task was risky, which is not what we want to portray here. In help articles, the task, "applying an upstream update", is what the doc should be about, and the contextual information is not as important and shouldn't be featured at the top. It helps give the user the steps they need to complete the task and if they want to read additional information, it's there (but not as prominent).

Comment (rachelwhitton) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: @nataliejeremy I see what you mean about keeping info needed to complete the task at the top and it being prominent over contextual info. Thanks for the info and insight!

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 2]
```diff
@@ -6,22 +6,6 @@ category:
   - drupal
 keywords: upstream, update upstream, apply updates, apply update, update core, update plugin, update module, update theme, update distribution, distribution, deploy update, deploy updates, update, updates, security update, apply security update, patch
 ---
-Core updates appear on your Dashboard after review and testing by our team, usually within a week of release. These updates appear in your code workspace beneath the Connection Mode bar when they are available.
```
Comment (bmackinney) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: My only rec is to keep a line of text like this one above the first H2.

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 3]
```diff
@@ -6,22 +6,6 @@ category:
...
-Core updates for alternate distributions (Open Atrium, Commerce Kickstart, etc.) are initiated by the maintainer, not Pantheon. Please contact them directly regarding expected updates.
-
-<div class="alert alert-danger" role="alert"><h4>Warning</h4>
-Only use the one-click updates on the Dashboard to update your site's core. Do not update core using Drush or WP-CLI; you will overwrite your core.<br /><br />
-
-You cannot unpack a tarball from Drupal.org or WordPress.org; this will overwrite the core's auto-configuration feature and your site will lose its database connection.</div>
-
-If you have overwritten core, see [Undo Git Commits](/docs/articles/local/undo-git-commits-like-overwriting-drupal-core).
 
 ## Preparation
```
Comment (bmackinney) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: Let's move this to the end, and call it something else.

File: source/docs/articles/sites/code/applying-upstream-updates.md [hunk 4]
```diff
@@ -32,6 +16,8 @@ If you have overwritten core, see [Undo Git Commits](/docs/articles/local/undo-g
 
 ## Apply a Core Update
```
Comment (bmackinney) [inline source/docs/articles/sites/code/applying-upstream-updates.md]: this line can be removed.

**Comment by ari-gold** [comment]
@nataliejeremy Can we remove/move the preparation section? Would be awesome if the image is above the fold/the first you thing you see.

**Comment by nataliejeremy** [comment]
@ari-gold We were just working on that :) Check it out now and let me know what you think.

**Comment by ari-gold** [comment]
Yay! Scariness factor decreased by 1000!

+1000

**Pull request merged**